src/client/texturesource.cpp

```diff
@@ -217,16 +190,17 @@ TextureSource::TextureSource()
 	m_main_thread = std::this_thread::get_id();
 
 	// Add a NULL TextureInfo as the first index, named ""
-	m_textureinfo_cache.emplace_back("");
+	m_textureinfo_cache.emplace_back(std::move(TextureInfo{""}));
```

Comment:
the move here is pointless (it is already an [xvalue](https://en.cppreference.com/w/cpp/language/value_category))

Reply:
Fixed.